packages/core/src/App/Containers/Layout/header/dtrader-header.jsx

```diff
@@ -100,114 +100,116 @@ const DTraderHeader = ({
+                    'header--is-disabled': is_app_disabled || is_route_modal_on,
+                    'header--is-hidden': platforms[platform],
+                })}
+            >
+                <div className='header__menu-items'>
+                    <div className='header__menu-left'>
+                        <DesktopWrapper>
+                            <PlatformSwitcher
+                                app_routing_history={app_routing_history}
+                                platform_config={filterPlatformsForClients(platform_config)}
+                                is_pre_appstore={is_pre_appstore}
+                            />
+                        </DesktopWrapper>
+                        <MobileWrapper>
+                            <ToggleMenuDrawer
+                                changeCurrentLanguage={changeCurrentLanguage}
+                                ref={toggle_menu_drawer_ref}
+                                should_allow_authentication={should_allow_authentication}
+                                account_status={account_status}
+                                enableApp={enableApp}
+                                disableApp={disableApp}
+                                location={location}
+                                logoutClient={logoutClient}
+                                is_dark_mode={is_dark_mode}
+                                is_logged_in={is_logged_in}
+                                is_p2p_enabled={is_p2p_enabled}
+                                is_payment_agent_transfer_visible={is_payment_agent_transfer_visible}
+                                is_onramp_tab_visible={is_onramp_tab_visible}
+                                is_payment_agent_visible={is_payment_agent_visible}
+                                is_account_transfer_visible={is_account_transfer_visible}
+                                is_virtual={is_virtual}
+                                is_pre_appstore={is_pre_appstore}
+                                toggleTheme={setDarkMode}
+                                platform_header={getPlatformInformation(app_routing_history).header}
+                                platform_switcher={
+                                    <PlatformSwitcher
+                                        app_routing_history={app_routing_history}
+                                        is_mobile
+                                        platform_config={filterPlatformsForClients(platform_config)}
+                                        toggleDrawer={toggle_menu_drawer_ref.current?.toggleDrawer}
+                                        is_pre_appstore={is_pre_appstore}
+                                    />
+                                }
+                                is_social_signup={is_social_signup}
+                                setIsPreAppStore={setIsPreAppStore}
+                            />
+                            {header_extension && is_logged_in && (
```

Comment:
'is_logged_in' here does not make senes anymore since you're checking at the top level wrapper.

Reply:
yeah it should be removed from here as its already being checked at top level wrapper.

Reply:
Why you are adding condition to the header? we need to check logged_in for trader's hub page

Reply:
Refactored the code as per discussion